packages/open-next/src/overrides/wrappers/cloudflare.ts

```diff
@@ -6,12 +6,20 @@ import type {
 
 import type { MiddlewareOutputEvent } from "../../core/routingHandler";
 
+const cfPropNameToHeaderName = {
+  city: "x-vercel-ip-city",
```

Comment:
If we decide to change the headers in `routingHandler` then it would make sense to rename those as `x-open-next`